Applications/DataExplorer/DataView/MshModel.cpp

```diff
@@ -63,8 +62,8 @@ void MshModel::addMeshObject(const MeshLib::Mesh* mesh)
 	QList<QVariant> meshData;
 	meshData << display_name << "";
 	MshItem* newMesh = new MshItem(meshData, _rootItem, mesh);
-	if (newMesh->vtkSource())
-		newMesh->vtkSource()->SetName(display_name);
+	//if (newMesh->vtkSource())
+	//	newMesh->vtkSource()->SetName(display_name);
```

Comment:
Please add a comment why this is not working or not executed. (or remove entirely).